sig-contributor-experience/charter.md

```diff
@@ -0,0 +1,59 @@
+# Contributor Experience Special Interest Group Charter
+*checking in for comments and will add TODOs at bottom along with feedback that comes in*
+
+The Contributor Experience Special Interest Group is dedicated to ensuring that Kubernetes contributors have a consistent workflow and great experience within the ecosystem. Developing and sustaining a healthy community of contributors is critical to scaling the project and growing the folks who build it. We champion the needs of these contributors by building programs and processes that address any gaps, learning and development, reward and recognize positive actions, and facilitiate project wide communications. 
+
+## Leads:
+Garrett Rodrigues  
+Paris Pittman  
+Elsie Phillips   
+
+## Processes and Responsibilities
+* Leads run weekly meetings in accordance with [SIG governance procedures](sig-governance.md)
+* Leads run quarterly planning meetings where we decide goals for the next release, to be published in the SIG’s [README](README.md)  
+* SIG Leads plan at least one annual face to face Contributor Experience SIG meeting proposed via mailing list and announced via slack and other channels
```

Comment:
Nit.. The line is missing period at the end. I notice inconsistency in using line ending period..let's be consistent :-).